tests/api/Jalasoft.TeamUp.Resumes.API.Tests/GetResumeByIdTests.cs

```diff
@@ -46,7 +46,7 @@ public void GetResume_Returns_ThrowException404()
         {
```

Comment:
please change the name from GetResume_Returns_ThrowException404 to GetResume_Returns_NotFound

Reply:
done